src/components/transmuters/EthTransmuterLooper.tsx

```diff
@@ -0,0 +1,652 @@
+  const { data: transmuterLooperContractDynamicState } = useReadContracts({
+    scopeKey: ScopeKeys.TransmuterLooperReported,
+    allowFailure: false,
+    contracts: [
+      {
+        // Get the timestamp at which all profits will unlock.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "fullProfitUnlockDate",
+        chainId: chain.id,
+      },
+      {
+        // Get status on whether the strategy is shut down.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "isShutdown",
+        chainId: chain.id,
+      },
+      {
+        // Get the timestamp of the last report
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "lastReport",
+        chainId: chain.id,
+      },
+      {
+        // Get the current balance in y shares of the address or account.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "balanceOf",
+        chainId: chain.id,
+        args: [address!],
+      },
+      {
+        // Get the price in asset per share
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "pricePerShare",
+        chainId: chain.id,
+      },
+      {
+        // Gets the current time profits are set to unlock over as uint.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "profitMaxUnlockTime",
+        chainId: chain.id,
+      },
+      {
+        // Gets the current time profits are set to unlock over as uint.
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "profitUnlockingRate",
+        chainId: chain.id,
+      },
+      {
+        // Gets the total assets deposited
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "totalAssets",
+        chainId: chain.id,
+      },
+      {
+        // Gets the total shares outstanding
+        address: TRANSMUTER_LOOPER_ADDRESS,
+        abi: tokenizedStrategyAbi,
+        functionName: "totalSupply",
+        chainId: chain.id,
+      },
+    ],
+    query: {
+      select: ([
+        fullProfitUnlockDate,
+        isShutdown,
+        lastReport,
+        balanceOf,
+        pricePerShare,
+        profitMaxUnlockTime,
+        profitUnlockingRate,
+        totalAssets,
+        totalSupply,
+      ]) => {
+        return [
+          BigInt(fullProfitUnlockDate as number),
+          isShutdown,
+          BigInt(lastReport as number),
+          BigInt(balanceOf as number),
+          BigInt(pricePerShare as number),
+          BigInt(profitMaxUnlockTime as number),
+          BigInt(profitUnlockingRate as number),
+          BigInt(totalAssets as number),
+          BigInt(totalSupply as number),
+        ];
+      },
+    },
+  });
```

Comment:
when you got this `transmuterLooperContractDynamicState`, you can destruct it right after the hook. this is cheap calculation and will always be in sync with current state (because transmuterLooperContractDynamicState is a state variable).

```jsx
 const [
    fullProfitUnlockDate,
    isShutdown,
    lastReport,
    balanceOf,
    pricePerShare,
    profitMaxUnlockTime,
    profitUnlockingRate,
    totalAssets,
    totalSupply,
  ] = transmuterLooperContractDynamicState ?? [];
```